pandas/core/groupby.py

```diff
@@ -1026,6 +1026,22 @@ def mean(self, *args, **kwargs):
 
     @Substitution(name='groupby')
     @Appender(_doc_template)
+    def quantile(self, *args, **kwargs):
```

Comment:
not sure what you are doing here, this doesn't affect ``.resample``.

but more importantly w/o tests it is impossible to tell.